NEWS.txt

```diff
@@ -1,5 +1,12 @@
 NEWS
 
+2.1.0
+
+- Now supporting Redis as a cache adapter
```

Comment:
Please don't repeat the notes in CHANGES. This is mostly for code changes needed to make the new version work with their existing code.